### ISSUE TITLE
Trouble with installation

### ISSUE THREAD

**Issue by mvinyard** [aertslab/cisTopic#42]
[in]:
```
devtools::install_github("aertslab/cisTopic")
```

[out]:
```
Downloading GitHub repo aertslab/cisTopic@master


Skipping 15 packages ahead of CRAN: GenomicRanges, S4Vectors, SummarizedExperiment, BiocGenerics, IRanges, XVector, GenomeInfoDb, zlibbioc, GenomicAlignments, Biobase, annotate, AnnotationDbi, DelayedArray, BiocParallel, GenomeInfoDbData

✔  checking for file ‘/tmp/RtmpktT8Ub/remotes2f89111f2a0/aertslab-cisTopic-8fd1432/DESCRIPTION’
─  preparing ‘cisTopic’:
✔  checking DESCRIPTION meta-information
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  looking to see if a ‘data/datalist’ file should be added
─  building ‘cisTopic_0.3.0.tar.gz’ (7.7s)
   

Installing package into ‘/home/mvinyard/R/x86_64-pc-linux-gnu-library/3.6’
(as ‘lib’ is unspecified)

Error: Failed to install 'cisTopic' from GitHub:
  (converted from warning) installation of package ‘/tmp/RtmpktT8Ub/file2f892524899b/cisTopic_0.3.0.tar.gz’ had non-zero exit status
Traceback:

1. devtools::install_github("aertslab/cisTopic")
2. pkgbuild::with_build_tools({
 .     ellipsis::check_dots_used(action = getOption("devtools.ellipsis_action", 
 .         rlang::warn))
 .     {
 .         remotes <- lapply(repo, github_remote, ref = ref, subdir = subdir, 
 .             auth_token = auth_token, host = host)
 .         install_remotes(remotes, auth_token = auth_token, host = host, 
 .             dependencies = dependencies, upgrade = upgrade, force = force, 
 .             quiet = quiet, build = build, build_opts = build_opts, 
 .             build_manual = build_manual, build_vignettes = build_vignettes, 
 .             repos = repos, type = type, ...)
 .     }
 . }, required = FALSE)
3. install_remotes(remotes, auth_token = auth_token, host = host, 
 .     dependencies = dependencies, upgrade = upgrade, force = force, 
 .     quiet = quiet, build = build, build_opts = build_opts, build_manual = build_manual, 
 .     build_vignettes = build_vignettes, repos = repos, type = type, 
 .     ...)
4. tryCatch(res[[i]] <- install_remote(remotes[[i]], ...), error = function(e) {
 .     stop(remote_install_error(remotes[[i]], e))
 . })
5. tryCatchList(expr, classes, parentenv, handlers)
6. tryCatchOne(expr, names, parentenv, handlers[[1L]])
7. value[[3L]](cond)
```

**Comment by mvinyard** [comment]
Not sure what's causing the issue - using R 3.6.2

**Comment by mvinyard** [comment]
I also output the `sessionInfo()` for more context, if that helps:

```
R version 3.6.2 (2019-12-12)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 18.04.3 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.7.1
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.7.1

locale:
 [1] LC_CTYPE=C.UTF-8       LC_NUMERIC=C           LC_TIME=C.UTF-8       
 [4] LC_COLLATE=C.UTF-8     LC_MONETARY=C.UTF-8    LC_MESSAGES=C.UTF-8   
 [7] LC_PAPER=C.UTF-8       LC_NAME=C              LC_ADDRESS=C          
[10] LC_TELEPHONE=C         LC_MEASUREMENT=C.UTF-8 LC_IDENTIFICATION=C   

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3          BiocManager_1.30.10 pillar_1.4.3       
 [4] compiler_3.6.2      prettyunits_1.1.1   base64enc_0.1-3    
 [7] remotes_2.1.0       tools_3.6.2         testthat_2.3.1     
[10] digest_0.6.23       pkgbuild_1.0.6      uuid_0.1-2         
[13] pkgload_1.0.2       jsonlite_1.6.1      evaluate_0.14      
[16] memoise_1.1.0       rlang_0.4.4         IRdisplay_0.7.0    
[19] cli_2.0.1           curl_4.3            IRkernel_1.1       
[22] repr_1.1.0          withr_2.1.2         desc_1.2.0         
[25] fs_1.3.1            devtools_2.2.1      rprojroot_1.3-2    
[28] glue_1.3.1          R6_2.4.1            processx_3.4.2     
[31] fansi_0.4.1         sessioninfo_1.1.1   pbdZMQ_0.3-3       
[34] callr_3.4.1         magrittr_1.5        backports_1.1.5    
[37] ps_1.3.0            ellipsis_0.3.0      htmltools_0.4.0    
[40] usethis_1.5.1       assertthat_0.2.1    crayon_1.3.4       
```

**Comment by cbravo93** [comment]
Hi @mvinyard !

I am unable to replicate the error (also R 3.6.2, and devtools_2.2.2). Does it work downloading the package and installing from source?

C

**Comment by aselewa** [comment]
Has this been resolved? I have the issue with R 3.5.1.
